lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -1291,7 +1291,6 @@
     }
   ],
   "Accessibility": {
-    "version": "2.1.1",
```

Comment:
@connorjclark I talked to @patrickhulce offline about this. After changing the axe-audit selector, there were changes  for the artifacts. We noticed that there hasn't been an update for axe in some time. That's why there's a lot of diff on this file, so would need another review from you, what do you think?

Reply:
LGTM